R/removerowsandorcolumns.R

```diff
@@ -28,11 +28,11 @@ RemoveRowsAndOrColumns <- function(x,
     if (is.null(row.names.to.remove) && is.null(column.names.to.remove))
         return(x)
 
-    if(inherits(x, "list"))
+    if (inherits(x, "list"))
     {
         # Removing the elements from the list.
         if (!is.null(nm <- names(x)))
-            x <- RemoveAt(x, at = column.names.to.remove, ignore.case = TRUE, split = split)#x[RemoveCharacterElements(nm, column.names.to.remove)]
+            x <- RemoveAt(x, at = column.names.to.remove, ignore.case = TRUE, split = split)
```

Comment:
`flipU::RemoveAt` also uses `CopyAttributes`, so looks like we'll need a patch there as well.